singleuser/README.md

```diff
@@ -10,6 +10,23 @@ class to launch user notebook servers within docker containers.
 
 This particular server runs (within the container) as the `jupyter` user, with
 home directory at `/home/jupyter`, and the IPython example notebooks at
-`/home/jupyter/examples`. This home directory is *not* persistent by default;
-thus, this image should be used with temporary or demonstration JupyterHub
+`/home/jupyter/examples`.
+
+## Important note on persistence
+
+This home directory, `/home/jupyter`, is *not* persistent; thus,
+this image should be used with temporary or demonstration JupyterHub
```

Comment:
Maybe something along the lines of "not persistent by default, so some configuration is required unless it is to be used with temporary or demonstration JupyterHub deployments"